### ISSUE TITLE
Cannot associate EBS disks with AWS 'Nitro' instance types

### ISSUE THREAD

**Issue by GlennS** [NixOS/nixops#1064]
Nitro instances don't appear to be able to associate EBS disks correctly, because NixOps has a hardcoded requirement for disk labels begining with `/dev/xvd`.

This requirement is correct for some instances, but not all.

https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html

So, if I write:

```
let
  region = "ap-southeast-2";
  zone = "ap-southeast-2a";
  accessKeyId = "default";
{
  resources.ebsVolumes."my-ebs-volume" = {
    inherit region zone accessKeyId;
    size = 100; # Gigabytes
    volumeType = "gp2";
  };

  myEC2Instance = {resources, pkgs, ...}: {
    deployment = {
      targetEnv = "ec2";
      ec2 = {
        inherit region zone accessKeyId;
        instanceType = "c5.large";
      };
    };

    fileSystems."/somewhere" = {
      device = "/dev/nvme1n1";
      fsType = "ext4";
      label = "my-fs";
      ec2 = {
        disk = resources.ebsVolumes."my-ebs-volume";
      };
    };
  };
}
```

Then I get the error:

`/dev/nvme1n1 is not a valid EBS device name`

**Comment by amemni** [comment]
Support of NVMe was recently added by #965 , I think it should be released with 1.6.1.

**Comment by GlennS** [comment]
Great, thanks :)
